### DISCUSSION TITLE
blog/2023%EB%85%84%209%EC%9B%94%20%ED%9A%8C%EA%B3%A0

### DISCUSSION THREAD

**Discussion by giscus** [khj0426/HJ_Devlog#12] [General]
# blog/2023%EB%85%84%209%EC%9B%94%20%ED%9A%8C%EA%B3%A0

9월의 끝!

https://hj-devlog.vercel.app/blog/2023%EB%85%84%209%EC%9B%94%20%ED%9A%8C%EA%B3%A0

**Comment by oaoong** [comment]
효중님 덕분에 프로젝트 잘 마무리 된 것 같아요!! 감사합니다

**Reply by khj0426** [reply]
저도 개발하면서 도움 많이 받았습니다..ㅜㅜ 너무 감사드려요
